config/routes.rb

```diff
@@ -7,16 +7,19 @@
 
     resources :sessions, only: [:create]
     resources :merchants, only: [:new, :create]
+
+    get 'products' => 'products#index'
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   end
   namespace :customers do
-    get 'signup'  => 'customers#new' 
+    get 'signup'  => 'customers#new'
     resources :customers
 
     get 'login'  => 'sessions#new'
     post 'login' => 'sessions#create'
-    delete 'logout' => 'sessions#destroy'
+    get'/logout' => 'sessions#destroy'
```

Comment:
to było potrzebne? pewnie wystarczyło dodać do link_to gdzie ta akcja jest wywoływana `method: :delete`

Reply:
próbowałam z method: :delete i jeszcze na kilka różnych sposób wymusić by użył delete i stale po kliknięciu wołał że nie ma ścieżki do get...

Reply:
próbowałaś w taki sposób mniej więcej? 
`<%= link_to 'delete', user, method: :delete, data: {:confirm => "You sure?" } ,
                            :title => "Delete #{user.name}" %>`

Reply:
spróbuję jeszcze raz, sama albo na zajęciach pod okiem Oli :)